meta/runtime.yml

```
@@ -126,6 +126,14 @@ plugin_routing:
       deprecation:
         removal_version: 9.0.0
         warning_text: This module relies on the deprecated package pyrax.
+    rhn_channel:
+      deprecation:
+        removal_version: 10.0.0
+        warning_text: RHN is EOL; please contact us if still using this.
```

Who is "us" here? RedHat? Collection maintainers?

---

I guess "us" is the module/collection maintainers. How about writing instead:
```suggestion
        warning_text: RHN is EOL; please create an issue in the community.general repository if still using this.
```
(It also appears below.)

One thing I would probably also change: mention that more details can be found in the module documentation.
```suggestion
        warning_text: RHN is EOL; please create an issue in the community.general repository if still using this. More information can be found in the module documentation.
```